core/src/main/java/org/apache/calcite/schema/Function.java

```diff
@@ -39,6 +39,14 @@
    * @return Parameters; never null
    */
   List<FunctionParameter> getParameters();
+
+  /**
+   * Returns the function is var args or not.
+   * @return Boolean
+   */
+  default boolean isVarArgs() {
```

Comment:
default is not needed

Reply:
it is a default method in function interface, it could not be removed.